packages/voici/src/app.ts

```diff
@@ -207,7 +212,17 @@ export class VoiciApp extends JupyterFrontEnd<IShell> {
         });
 
         // Create Voila widget manager
-        const widgetManager = new VoiciWidgetManager(kernel, rendermime);
+        const widgetManager = new KernelWidgetManager(kernel, rendermime);
+        rendermime.removeMimeType(WIDGET_MIMETYPE);
+        rendermime.addFactory(
+          {
+            safe: false,
+            mimeTypes: [WIDGET_MIMETYPE],
+            createRenderer: (options) =>
+              new WidgetRenderer(options, widgetManager),
+          },
+          -10
+        );
```

Comment:
It's unclear to me why we need to do this? Why is the already registered mimetype renderer for widgets not sufficient?

Reply:
Right I am also unsure. This change mimics what is in https://github.com/voila-dashboards/voila/pull/1249. But maybe there is a way to simplify all this.

Reply:
It's possible we need it because the stock `@jupyter-widgets/jupyterlab-manager` is not able to render the widgets found on the page if they don't come from a notebook (or document context).